### ISSUE TITLE
Test the "Release It" CLI tool for GitHub releases

### ISSUE THREAD

**Issue by SDavisMedia** [SDavisMedia/grunt-bruh#6]
Hope it works well, bruh. https://www.npmjs.com/package/release-it

**Comment by SDavisMedia** [comment]
Grunt Release is pretty sweet. First pass was aborted due to warnings, though. 

<img width="460" alt="screenshot 2018-04-04 13 45 48" src="https://user-images.githubusercontent.com/2359131/38333812-817c5a22-380e-11e8-9c87-1f03c9f72940.png">

- [ ] Depending on issue, might have to install `grunt-force-task` to get through the release process.
- [ ] Grunt Release doesn't include a changelog generator, so it'll need to be combined with another task.

**Comment by SDavisMedia** [comment]
I think it does have a changelog tool, actually. Time to configure some options. 😈

**Comment by SDavisMedia** [comment]
Never mind, no changelog tool. But there are options that can be configured to run other grunt tasks as part of the release process. So I still have to find an independent changelog tool.

**Comment by SDavisMedia** [comment]
😍 https://www.npmjs.com/package/github-release-notes

**Comment by SDavisMedia** [comment]
Gotta work from `master` at this point. So many tests. 😓
